ui/src/pages/plan/rules/CreateRuleForm.test.tsx

```diff
@@ -1,6 +1,16 @@
 import React from 'react';
-import { render, fireEvent } from '@testing-library/react'
```

Comment:
There is a compile error happening for fireEvent. Before merging please see if you are getting a compile error aswell.

Reply:
Resolved.